libs/ui/select/brain/src/lib/brn-select.component.ts

```diff
@@ -155,6 +154,8 @@ export class BrnSelectComponent implements ControlValueAccessor, AfterContentIni
 	// eslint-disable-next-line @typescript-eslint/no-empty-function
 	private _onTouched = () => {};
 
+	private _shouldEmitValueChange = signal(false);
```

Comment:
Is a signal really needed here?

Reply:
I guess technically it does not need to be a signal since this value only ever changes if writeValue is triggered or we change value internally. Does it matter you think or would you prefer to just have it has a regular variable?

Reply:
I just stopped using signals everywhere. 
I think it is unnecessary overhead, even if the overhead of a signal is extremely small. 
If i is a signal, I will assume that there is something reacting on it.
These are just very small arguments and I am fine with keeping it a signal.